### DISCUSSION TITLE
Same session, multiple size windows

### DISCUSSION THREAD

**Discussion by losnappas** [zellij-org/zellij#2439] [General]
I'd like to have one session and attach to it from multiple different sized viewports. In zellij, as in tmux, it makes all the viewports the same size, overflowing on smaller and limiting size on bigger views.

Is it possible to have a feature to correctly size all of them, independent of each other? Or is that an impossibility?

**Comment by overby-me** [comment]
Any update on this?

**Comment by ethanjli** [comment]
This would be useful even just in the specific case that the different viewports are attached to different tabs of a session
